src/core/tranche/operator.sol

```diff
@@ -78,9 +73,9 @@ contract Operator is DSNote {
 
     function supply(address usr, uint currencyAmount) public note auth {
         require (supplyActive);
-        uint tokenAmount = slicer.calcSlice(currencyAmount);
+        uint tokenAmount = slicer.getSlice(currencyAmount);
```

Comment:
It is a bit confusing we have `getSlice` and `sliceOf` which sounds really similar but they are doing different things

Reply:
renamed sliceOf(address usr) to tokenBalance(address usr)